apps/server/src/modules/tool/controller/dto/request/external-tool-config.params.ts

```diff
@@ -0,0 +1,7 @@
+import { ToolConfigType } from '@src/modules/tool/interface/tool-config-type.enum';
+
+export abstract class ExternalToolConfigCreateParams {
+	abstract type: ToolConfigType;
+
+	abstract baseUrl: string;
+}
```

Comment:
Maybe it's an interface?

Reply:
We would like to keep it as an abstract class, see https://docs.dbildungscloud.de/pages/viewpage.action?spaceKey=N21P&title=Data+Model+and+Workflow

it was discussed that way in the architecture group, too :)